lib/debug/observatory_debugger.dart

```diff
@@ -473,12 +478,16 @@ class ObservatoryIsolate extends DebugIsolate {
       (libraryRef) => new ObservatoryLibrary._(libraryRef)).toList();
   }
 
-  void _suspend(bool value) {
-    if (!value) frames = null;
+  void _suspend(bool paused, bool pausedAtAsyncSuspension) {
```

Comment:
Perhaps used a named param for the 2nd param, just to avoid having call sites w/ a list of booleans (`_suspend(false, false)`) - can be hard to parse later.